docs/release-notes/7.17.asciidoc

```diff
@@ -5,6 +5,44 @@
 [[release-notes-7.17.6]]
 === 7.17.6
 
+[discrete]
+[[known-issue-7.17.6]]
+==== Known issues
+* In some situations, {elastic-endpoint} might change to a non-running state on Windows endpoints and cannot be restarted. {agent} will have an `Unhealthy` status when this happens (https://github.com/elastic/endpoint/issues/29[#29]).
```

Comment:
```suggestion
* In some situations, {elastic-endpoint} might change to a non-running state on Windows endpoints and fail to restart. {agent} will have an `Unhealthy` status when this happens (https://github.com/elastic/endpoint/issues/29[#29]).
```

Reply:
if you accept this suggestion you can apply it to the other version of release notes in this PR as well